### ISSUE TITLE
In "Preferences" to have the possibility to define a shortcut which will lanch/hide the FSearch Window

### ISSUE THREAD

**Issue by silviulaurentiu** [cboxdoerfer/fsearch#506]

**Comment by nekohayo** [comment]
In my view, this is the desktop environment's job, particularly if the app is not running. But even if it _is_ already running… if it's not handled by the DE, then it would be impossible for the app to do this under Wayland, because there are no such things as the ability for apps to set global keyboard shortcuts (at least currently).

I'd suggest closing this as out-of-scope/not-fixable.

**Comment by cboxdoerfer** [comment]
Yeah, launching an app with a custom shortcut is already possible with pretty much any desktop environment. The same applies to minimizing a window.

@silviulaurentiu or do you mean having one shortcut which toggles the visibility of FSearch?

**Comment by al0cam** [comment]
I would like to have the option to toggle the visibility of FSearch with one shortcut.
Otherwise I'd have to write a script which kills the process if it exists or something similar.
Maybe I am not aware of some toggle feature, however as far as I can see from the man page and help screen a toggle isn't present. 
Would be greatly appreciated.
I can help write it, just point me in the right direction.

**Comment by nekohayo** [comment]
Update regarding my previous comment above: there _is_ now a way to set global keyboard shortcuts in Wayland, though I don't know if this can be used to actually launch an app: https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.GlobalShortcuts.html
